ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -385,6 +397,90 @@ public class BottomSheetController: UIViewController, Shadowable {
     public var ambientShadow: CALayer?
     public var keyShadow: CALayer?
 
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        tokenSet.update(fluentTheme)
+    }
+
+    public override func viewDidLayoutSubviews() {
+        let newHeight = view.bounds.height
+        if currentRootViewHeight != newHeight && currentExpansionState == .transitioning {
+            // The view height has changed and we can't guarantee the animation target frame is valid anymore.
+            // Let's complete animations and cancel ongoing gestures to guarantee we end up in a good state.
+            completeAnimationsIfNeeded(skipToEnd: true)
+
+            if panGestureRecognizer.state != .possible {
+                panGestureRecognizer.state = .cancelled
+            }
+        }
+        currentRootViewHeight = newHeight
+
+        // In the transitioning state a pan gesture or an animator temporarily owns the sheet frame updates,
+        // so to avoid interfering we won't update the frame here.
+        if currentExpansionState != .transitioning {
+            bottomSheetView.frame = sheetFrame(offset: offset(for: currentExpansionState))
+            updateSheetLayoutGuideTopConstraint()
+            updateExpandedContentAlpha()
+            updateDimmingViewAlpha()
+            updateDimmingViewAccessibility()
+        }
+        collapsedHeightInSafeArea = view.safeAreaLayoutGuide.layoutFrame.maxY - offset(for: .collapsed)
+
+        updateAppearance()
+    }
+
+    public override func viewSafeAreaInsetsDidChange() {
+        super.viewSafeAreaInsetsDidChange()
+        completeAnimationsIfNeeded(skipToEnd: true)
+    }
+
+    public override func willTransition(to newCollection: UITraitCollection, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.willTransition(to: newCollection, with: coordinator)
+        completeAnimationsIfNeeded(skipToEnd: true)
+    }
+
+    public typealias TokenSetKeyType = BottomSheetTokenSet.Tokens
+    public var tokenSet: BottomSheetTokenSet = .init()
+
+    var tokenSetSink: AnyCancellable?
+    var fluentTheme: FluentTheme { return view.fluentTheme }
+
+    private func updateAppearance() {
+        updateBackgroundColor()
+        updateShadow()
+        updateCornerRadius()
+
+        super.viewDidLayoutSubviews()
```

Comment:
Was this meant to be called in viewDidLayoutSubviews?

Reply:
I was persisting it from the existing position in the updateShadow() method, but looking through the history, it just looks like a merge conflict mistake. I'll remove it!